### DISCUSSION TITLE
llama.cpp wrapper in  langchain.net?

### DISCUSSION THREAD

**Discussion by markat1** [tryAGI/LangChain#410] [Q&A] [answered]
Is there a llama.cpp wrapper in langchain.net? [example here in python langchain](https://python.langchain.com/v0.2/docs/integrations/providers/llamacpp/#embeddings )

**Comment by HavenDV** [comment, marked as answer]
We have a provider for https://github.com/SciSharp/LLamaSharp that provides bindings for llama.cpp, but its underlying maintainer hasn't updated it in a while and it may not work at the moment

**Reply by markat1** [reply]
okay I will try ask LLamaSharp for an update  - thanks!

**Reply by HavenDV** [reply]
I checked and it works, so I don't think any action is required.
